docs/dev/libvirt-howto.md

```diff
@@ -235,6 +235,12 @@ openshift-install destroy cluster
 
 If you did not compile with `libvirt_destroy`, you can use [`virsh-cleanup.sh`](../../scripts/maintenance/virsh-cleanup.sh), but note it will currently destroy *all* libvirt resources.
 
+Note that you almost certainly also want to clean up the installer
+state files too, including `auth/`, `terraform.tfstate` etc.  The
```

Comment:
We don't wrap docs at 80 columns. I know, I die a little inside as well. Can you pull all of this onto a single line?

Reply:
I also like one-sentence per line, like [this][1] for less churny `git blame`.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.1/style.md#one-sentence-per-line

Reply:
Resolved for one sentence per line in 491ff5a -> 6fa6c57.